Neos.Flow/Classes/ResourceManagement/Target/FileSystemSymlinkTarget.php

```diff
@@ -58,8 +57,9 @@ public function publishCollection(CollectionInterface $collection, callable $cal
      */
     protected function publishFile($sourceStream, $relativeTargetPathAndFilename)
     {
-        $pathInfo = UnicodeFunctions::pathinfo($relativeTargetPathAndFilename);
-        if (isset($pathInfo['extension']) && array_key_exists(strtolower($pathInfo['extension']), $this->excludedExtensions) && $this->excludedExtensions[strtolower($pathInfo['extension'])] === true) {
+        $pathInfo = pathinfo($relativeTargetPathAndFilename);
```

Comment:
Well, then it'd be `$extension = strtolower(pathinfo($relativeTargetPathAndFilename, PATHINFO_EXTENSION));` right away, because with a flag given, a string is returned.

But auto-merge was faster. ;)

Reply:
See https://github.com/neos/flow-development-collection/pull/2729